service/cloudconfigv2/cloud_config.go

```diff
@@ -52,60 +51,60 @@ func New(config Config) (*CloudConfig, error) {
 
 // NewMasterTemplate generates a new worker cloud config template and returns it
 // as a base64 encoded string.
-func (c *CloudConfig) NewMasterTemplate(customObject kvmtpr.CustomObject, certs certificatetpr.AssetsBundle, node clustertprspec.Node) (string, error) {
+func (c *CloudConfig) NewMasterTemplate(customObject v1alpha1.KVMConfig, certs certificatetpr.AssetsBundle, node v1alpha1.ClusterNode) (string, error) {
```

Comment:
The diff is not that nice but the templating/invocation the package is doing got nicer. We actually do not need any weird conditionals anymore. In fact, we never made real use of it because of our intermediate transitioning state. In the future we version based on the custom object within the resource router. The packages invoked then are already the ones we want to execute.